yt_dlp/extractor/archiveorg.py

```diff
@@ -945,3 +946,238 @@ def _real_extract(self, url):
+                                                 'gcc': user_country,
+                                                 'locale': 'en_US',
+                                             }, fatal=False)
+        if not inkey_data:
+            raise ExtractorError('This video was not archived', expected=True)
+        inkey = self._parse_json(inkey_data, video_id)
+
+        vod_id = traverse_obj(player_info, ('postDetail', 'post', 'officialVideo', 'vodId'))
+
+        vod_data = self._parse_json(self._download_wbm_page(f'https://apis.naver.com/rmcnmv/rmcnmv/vod/play/v2.0/{vod_id}', video_id,
+                                    note='Fetching vod data', query={
+                                        'key': inkey.get('inkey'),
+                                        'pid': 'rmcPlayer_16692457559726800',  # partially unix time and partially random. Fixed value used by archiveteam project
+                                        'sid': '2024',
+                                        'ver': '2.0',
+                                        'devt': 'html5_pc',
+                                        'doct': 'json',
+                                        'ptc': 'https',
+                                        'sptc': 'https',
+                                        'cpt': 'vtt',
+                                        'ctls': '%7B%22visible%22%3A%7B%22fullscreen%22%3Atrue%2C%22logo%22%3Afalse%2C%22playbackRate%22%3Afalse%2C%22scrap%22%3Afalse%2C%22playCount%22%3Atrue%2C%22commentCount%22%3Atrue%2C%22title%22%3Atrue%2C%22writer%22%3Atrue%2C%22expand%22%3Afalse%2C%22subtitles%22%3Atrue%2C%22thumbnails%22%3Atrue%2C%22quality%22%3Atrue%2C%22setting%22%3Atrue%2C%22script%22%3Afalse%2C%22logoDimmed%22%3Atrue%2C%22badge%22%3Atrue%2C%22seekingTime%22%3Atrue%2C%22muted%22%3Atrue%2C%22muteButton%22%3Afalse%2C%22viewerNotice%22%3Afalse%2C%22linkCount%22%3Afalse%2C%22createTime%22%3Afalse%2C%22thumbnail%22%3Atrue%7D%2C%22clicked%22%3A%7B%22expand%22%3Afalse%2C%22subtitles%22%3Afalse%7D%7D',
+                                        'pv': '4.26.9',
+                                        'dr': '1920x1080',
+                                        'cpl': 'en_US',
+                                        'lc': 'en_US',
+                                        'adi': '%5B%7B%22type%22%3A%22pre%22%2C%22exposure%22%3Afalse%2C%22replayExposure%22%3Afalse%7D%5D',
+                                        'adu': '%2F',
+                                        'videoId': vod_id,
+                                        'cc': user_country,
+                                    }), video_id)
+
+        formats = []
+
+        if len(vod_data.get('streams', [])) > 1:
+            self.report_warning('Multiple streams found. Only the first stream will be downloaded.')
+        stream = traverse_obj(vod_data, ('streams', 0), {})
+
+        max_stream = max(stream.get('videos', []), key=lambda v: traverse_obj(v, ('bitrate', 'video')), default=None)
```

Comment:
Why not fetch all formats?

Reply:
Only the highest bitrate format got archived. See https://github.com/ArchiveTeam/vlive-grab/blob/master/vlive.lua#L451-L457